main/src/cgeo/geocaching/maps/mapsforge/v6/NewMap.java

```diff
@@ -279,6 +280,15 @@ public void onCreate(final Bundle savedInstanceState) {
         mapView.getMapScaleBar().setVisible(true);
         mapView.setBuiltInZoomControls(true);
 
+        // style zoom controls
+        final MapZoomControls zoomControls = mapView.getMapZoomControls();
+        zoomControls.setZoomControlsOrientation(MapZoomControls.Orientation.VERTICAL_IN_OUT);
+        zoomControls.setZoomInResource(R.drawable.zoomin);
+        zoomControls.setZoomOutResource(R.drawable.zoomout);
+        final int padding = ViewUtils.dpToPixel(10.0f);
+        zoomControls.setPadding(padding, padding, padding, padding);
+        zoomControls.setAutoHide(false);
```

Comment:
Should we really disable `AutoHide`? I know that's the way how GMap behaves, but is it really preferable?

Reply:
To be discussed...

Pro: consistency to GM; also I find it kind of annoying having to hit the map twice to zoom. Since new zoom controls are smaller than the old one, hiding too much map space should not be a problem anymore.

Con: Some people may want to see every inch of map space (due to extremely small screens etc.)

Reply:
Let's stay with `setAutoHide(false)` for now and wait for user feedback.

Reply:
I also would prefer auto hiding simply to avoid the problem, that I cannot click a cache below the zoom controls.

Reply:
> that I cannot click a cache below the zoom controls.

You can simply move the map to reach the cache...

Reply:
or the other way around on GMaps: 
I cannot zoom but it opens the cache....but anyway you are right: Both kind of a luxury problem.